CHANGELOG.md

```diff
@@ -1,10 +1,30 @@
 # Fluidcontent Change log
 
+upcoming
+------------------
+
+- :exclamation: Support of [TYPO3 6.2 dropped](https://github.com/FluidTYPO3/fluidcontent/commit/f1df9a4b7ee563335c8f1feeb7925e6f820a7f19)
```

Comment:
@NamelessCoder you mean, there should be now 2 **upcoming** sections: first - for 6.2 and changelog for legacy branches should be there, second section - for 7.high_five_shark version with own changelog, right?